frontend/packages/console-app/src/components/quick-starts/QuickStartDrawerAsync.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import { AsyncComponent } from '@console/internal/components/utils';
+
+const QuickStartDrawerAsync: React.FC = ({ children, ...props }) => (
+  <AsyncComponent loader={() => import('./QuickStartDrawer').then((c) => c.default)} {...props}>
```

Comment:
Can we use the same chunk name for all quick start async components? I think that will bundle the chunks together.
@vojtechszocs @spadgett do you know?
```suggestion
  <AsyncComponent loader={() => import('./QuickStartDrawer' /* webpackChunkName: "quick-start" */).then((c) => c.default)} {...props}>
```